app/views/clients/viewclient.html

```diff
@@ -965,11 +965,13 @@ <h4><strong>{{client.displayName}}</strong></h4>
 							<td>{{clientdocument.fileName}}</td>
 							<td>
 								<a target="_blank" href="{{hostUrl}}{{clientdocument.docUrl}}"><i class="fa fa-cloud-download"></i></a>&nbsp;
+								<a ng-show="clientdocument.fileIsImage" ng-click="previewDocument(clientdocument.docUrl,clientdocument.fileName)"><i class="fa fa-eye"></i></a>
 								<a ng-click="deleteDocument(clientdocument.id,$index)"><i class="fa fa-times"></i></a>
 							</td>
 						</tr>
 						</tbody>
 					</table>
+					<object height="400px" width="400px" ng-show="preview" data="{{fileUrl}}" type="{{fileType}}"></object>
```

Comment:
Sorry about making you do changes again and again, I forgot to tell about making this center aligned my bad. Also the Image size could be width="70%" height="60%" It looked kind of better to me.

I noticed that `.jpeg` files don't have preview so I think they should be included too.

Sorry for the late comments.

You could make the changes then do 

1. `git add` - Add changes
2. `git commit --amend` - Overwrite previous commit
3. `git push -f origin upload-preview

Reply:
Ok, made the changes. Also align:center didn't work, so I used padding-left: 25% for center align.